x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -2308,3 +2308,15 @@ message:keyword
 foo ( bar
 // end::rlikeEscapingTripleQuotes-result[]
 ;
+
+mvStringEquals
```

Comment:
Do we have a test for an literal over `ignored_above` chars + MV?

Reply:
Literals don't have `ignore_above`.